lib/get-seam.ts

```diff
@@ -1,25 +1,37 @@
-import { SeamHttp } from "@seamapi/http/connect"
+import { SeamHttp, SeamHttpMultiWorkspace } from "@seamapi/http/connect"
 import { getConfigStore } from "./get-config-store"
 import { getServer } from "./get-server"
 
-export const getSeam = () => {
+export const getSeam = async (): Promise<SeamHttp> => {
   const config = getConfigStore()
 
   const token = config.get(`${getServer()}.pat`)
+
   const token_type = token.startsWith("seam_at")
```

Comment:
this would be nice to have in the module fwiw

Reply:
Whoops, I think we have the same idea: https://github.com/seamapi/seam-cli/pull/46/files#r1434516504